test/test_scanner.py

```diff
@@ -64,19 +64,6 @@ def scan_file(self, filename):
         """ Run the scanner on a file """
         return self.scanner.scan_file(os.path.join(BINARIES_PATH, filename))
 
-    def test_vendor_package_pairs(self):
```

Comment:
Does this test need to be changed for it to make sense with the new scanner setup?  It looks like it's checking to make sure the vendor package pairs get loaded, but I'm not sure if we still need this test with the refactoring.   @pdxjohnny, do you remember why this test exists?

Reply:
We don't need this test now because we are storing vendor -package pair now as class variable of Checker. In the previous case we were loading vendor-package pair from the doc-string. So, that test was necessary to check `load-vendor-package` method work.

Reply:
works for me!